src/components/breadcrumbs/template.tsx

```diff
@@ -82,8 +82,12 @@ const Breadcrumbs = ({
   const fcString = JSON.stringify(filteredCrumbs)
 
   return (
-    <div className="vads-grid-container">
-      <va-breadcrumbs class="row" wrapping breadcrumb-list={fcString} />
+    <div className="vads-grid-container" style={{ minHeight: '5.125rem' }}>
```

Comment:
The mobile breadcrumbs height was ~`82px` for me. This translates to `5.125rem`. I'm using `rem` here so the font size increase also gets accounted for.